lalrpop/src/build/mod.rs

```diff
@@ -29,32 +31,56 @@ mod fake_term;
 
 use self::fake_term::FakeTerminal;
 
+pub fn process() -> io::Result<()> {
+    let session = Session::new();
+
+    let src_dir = try!(current_dir()).join("src");
+
+    // The environment variable OUT_DIR is set by cargo, and specifies
+    // a directory where generated code should be put.
+    process_dir(&session, &src_dir, Path::new(&env::var("OUT_DIR").unwrap()))
+}
+
 pub fn process_root() -> io::Result<()> {
     let session = Session::new();
-    process_dir(&session, try!(current_dir()))
+    let cwd = try!(current_dir());
+    process_dir(&session, &cwd, &cwd)
```

Comment:
Hmm, I feel like we should still put stuff into the OUT_DIR, or at least we should do so if it is located under `src`. Eventually I guess we might want a configuration option here.